Source/Core/CesiumTerrainProvider.js

```diff
@@ -198,6 +192,31 @@ import TileProviderError from './TileProviderError.js';
             var maxZoom = data.maxzoom;
             overallMaxZoom = Math.max(overallMaxZoom, maxZoom);
             // Keeps track of which of the availablity containing tiles have been loaded
+
+            if (!data.tilingScheme || data.tilingScheme === 'GlobalGeodetic') {
```

Comment:
Rather than add a new property, we should use the `projection` property already found in the layer.json. `WebMercatorTilingScheme` should be specified with `"EPSG:3857"`.

Reply:
Ok that sounds good, should it default to the existing behaviour in all other cases?

Reply:
I think the general logic you have here is good. If it's EPSG:3857, do web mercator. If it's undefined or EPSG:4326, do geographic. If it's something else, complain.